mate-api.md

```diff
@@ -324,6 +328,80 @@ type CourseUser {
 }
 ```
 
+#### 2.3. Use @Query, @Mutation, @Child decorators while creating resolvers
+>❓ It's a part of moving to Dependency Injection pattern in API, and decorators are more self explainable. Now `makeResolver` and `makeAuthResolver` functions are deprecated
+
+```typescript
+// ❌ bad
+export const chatLessonResolver = makeAuthResolver<
+  // ... types go here
+>(
+  GetChatLessonUseCase,
+  (_, chat) => ({ chatId: chat.id }),
+);
+
+
+// ✅ good
+@Child<
+  // ... types go here
+  >({
+    Handler: GetChatLessonUseCase,
+    auth: true,
+    mapOptions: (args, parent) => ({
+      chatId: chat.id,
+    }),
+  })
+
+export class getCareerTestQuestionsResolver {}
```

Comment:
See next example :harold:
```suggestion
export class GetCareerTestQuestionsResolver {}
```

Reply:
cc @ivan-kadykalo Seems it is your resolver. Pay attention to the resolvers name convention